backend/dataall/core/environment/cdk/pivot_role_core_policies/s3.py

```diff
@@ -18,6 +18,7 @@ def get_statements(self):
                     's3:ListAllMyBuckets',
                     's3:GetBucketLocation',
                     's3:PutBucketTagging'
+                    's3:GetEncryptionConfiguration'
```

Comment:
We should restrict this permission to dataset buckets only. For data.all managed buckets this permission is already included in the Get*. For imported buckets I have added it to the datasets/cdk/pivot role policy